lib/common_test/src/ct_framework.erl

```diff
@@ -1416,7 +1416,7 @@ report(What,Data) ->
 	    %% top level test index page needs to be refreshed
 	    TestName = filename:basename(?val(topdir, Data), ".logs"),
 	    RunDir = ?val(rundir, Data),
-	    _ = ct_logs:make_all_suites_index({TestName,RunDir}),
+	    _ = ct_logs:make_all_suites_index({TestName,RunDir},unknown),
 	    ok;
```

Comment:
would it make code less ugly if `unknown` is a default when `ct_logs:make_all_suites_index/1` ?
maybe ct_framework module can be left alone, not being aware about stylesheet?

Reply:
No leave it.